portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <script src="https://kit.fontawesome.com/848c0f5fe3.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Akira Tong: my journey </title>
+</head>
+<body>
+    <header>
+        <a href="index.html" class="logo">AKKKKIRA</a>
+        <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="index.html#sec">About Me</a></li>
+            <li><a href="work.html">Work (Coding && Music)</a></li>
+            <li><a href="blog.html" class="active">Blog</a></li>
+        </ul>
+    </header>
+    <div>
+        <br>
+        <br><br><br>
+        <br>
+    </div>
+    <div>
+    <h1 style="color: white" id="text1">my blog</h1>
+    <p style="color: white" id="text2">here I will share my journey, not only as a student, but also as a producer and a human.</p>
+    </div>
+    <div id="blog">
+        <h2 id="text2">ranDoM thOugHTs</h2>
+        <ul>
+            <li><a href="blogs/what-if-die.html">??? 01.</a></li>
```

Comment:
At least w/in Google, I think the convention would be to add the destination of the links at the same time the links were added. You could do that by creating dummy pages to be filled in later @ the destinations or by creating the framework of the page, then adding the links later. No reason to change things now, just food for thought going forward.